ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2800,6 +2800,55 @@ def test_numpy_instance_array_wrap__(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="take",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"), num_arrays=1
+    ),
+)
+@given(
+    indices=st.one_of(
+        st.integers(),
+        st.lists(st.integers()),
+        st.tuples(st.integers()),
+        st.arrays(dtype=np.int64, shape=st.tuples(st.integers())),
+    ),
+    axis=st.one_of(st.none(), st.integers()),
+    mode=st.one_of(st.just("raise"), st.just("wrap"), st.just("clip")),
+)
+def test_numpy_instance_take(
+    dtype_and_x,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    frontend,
+    indices,
+    axis,
+    mode,
+):
+    input_dtypes, xs = dtype_and_x
+
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtypes,
+        init_all_as_kwargs_np={
+            "object": xs[0],
+        },
+        method_input_dtypes=[np.int64],
+        method_all_as_kwargs_np={
+            "indices": indices,
+            "axis": axis,
+            "out": None,
```

Comment:
`out` is handled automatically, can you remove it?

Reply:
I have changed it. now it is not handled automatically